Sources/Charcoal/Filters/FreeText/FreeTextFilterViewController.swift

```diff
@@ -223,13 +228,22 @@ extension FreeTextFilterViewController: UISearchBarDelegate {
 
 private extension FreeTextFilterViewController {
     func returnToSuperView() {
+        if isFreeTextOnlySelected {
+            dismissView()
```

Comment:
Can we move `isFreeTextOnlySelected = false` here? `dismissView()` could potentially be used in another context than for "free text only", but then it shouldn't necessarily reset this unrelated var.